.github/workflows/publish.yml

```
@@ -0,0 +1,114 @@
+        run: npm install -g npm@latest
+
+      - run: yarn
+
+      - name: build
+        run: yarn build
+
+      - name: Publish subgraph-query
+        uses: ./.github/actions/create-release
+        with:
+          package-path: .
+          repo-token: ${{ secrets.REPO_TOKEN
```

actionlint (1.7.9)

unexpected EOF while lexing expression

(expression)